iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -157,13 +157,29 @@ LogicalResult verifyDoubleTilingExpertPassPipelineConfig(
     }
   }
 
+  int numLoops = interfaceOp.getNumLoops();
+  SmallVector<int64_t> secondLevelTileSizes = loweringConfig.getTileSizeVals(
+      static_cast<unsigned>(TilingLevel::L1Tiles));
+  if (secondLevelTileSizes.size() != numLoops) {
```

Comment:
Based on overall comment on the PR. We need to check that the tile sizes are specified only for the parallel loops.

Reply:
Do we get parallel loops by `PartitionableLoopsInterface::getPartitionableLoops` method?
The method takes `maxNumPartitionedLoops` argument. In this case, do we pass INF? (or maybe we can add another method to the interface)

Reply:
You should be able to use this https://github.com/google/iree/blob/15fce0a6c29b7eb810e6ad36706293dcb855148d/iree/compiler/Codegen/Utils/Utils.h#L21

(and yes you should be using the `getPartitionableLoops` method (at least for now).

Reply:
getPartitionableLoops is a bit different from what I thought. It also drops unit dims and makes logic a bit complicated, which is fine I think.